examples/with-imagekit/.env.local.example

```diff
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_PUBLIC_KEY= 
+NEXT_PUBLIC_URL_ENDPOINT=
+IMAGEKIT_FOLDER=
+PRIVATE_KEY=
```

Comment:
Can we rename this to `IMAGEKIT_PRIVATE_KEY`?

Reply:
done